ckb-vm-syscall-tracer/src/lib.rs

```diff
@@ -1018,6 +1018,221 @@ enum PartialLocator {
     Spawn(BinaryLocator),
 }
 
+enum PartialVmCreate {
+    Exec(CollectorKey),
+    Spawn(CollectorKey),
+}
+
+#[derive(Default, Clone)]
+pub struct VmCreateCollector {
+    pending: Arc<Mutex<Option<PartialVmCreate>>>,
+    data: Arc<Mutex<HashMap<CollectorKey, Vec<CollectorKey>>>>,
+    generation_tracker: GenerationTracker,
+}
+
+impl Collector for VmCreateCollector {
+    type Trace = Vec<CollectorKey>;
```

Comment:
Let's just discuss this `Trace` type, assuming I want to use this collector to build a VM invocation tree for a particular CKB script, how should I use the data structure here?

And just to add more: yes we will need an answer here, but it would be best if the answer to this question, is also an example for current crate.

Reply:
A practical example is: https://github.com/mohanson-fork/ckb-standalone-debugger/blob/tracer/ckb-debugger/src/main.rs#L466-L518

```sh
$ ckb-debugger --mode full --tx-file examples/spawn.json
```

It will output:

```
Spawn tree: 0                    cell_dep[0][0..101624]
Spawn tree: └── 1                cell_dep[1][0..100960]
```